packages/calcite-components/src/components/sheet/sheet.scss

```diff
@@ -11,12 +11,12 @@
  * @prop --calcite-sheet-resize-icon-color: Specifies the text color of the resize handle.
  *
  * @prop --calcite-sheet-scrim-background: Specifies the background color of the sheet scrim.
- * @prop --calcite-sheet-width: When `position` is `"inline-start"` or `"inline-end"`, specifies the width of the component.
- * @prop --calcite-sheet-max-width: When `position` is `"inline-start"` or `"inline-end"`, specifies the maximum width of the component.
- * @prop --calcite-sheet-min-width: When `position` is `"inline-start"` or `"inline-end"`, specifies the minimum width of the component.
- * @prop --calcite-sheet-height:  When `position` is `"block-start"` or `"block-end"`, specifies the height of the component.
- * @prop --calcite-sheet-max-height:  When `position` is `"block-start"` or `"block-end"`, specifies the maximum height of the component.
- * @prop --calcite-sheet-min-height:  When `position` is `"block-start"` or `"block-end"`, specifies the minimum height of the component.
+ * @prop --calcite-sheet-width: Specifies the width of the component when `position` is `"inline-start"` or `"inline-end"`.
+ * @prop --calcite-sheet-max-width: Specifies the maximum width of the component when `position` is `"inline-start"` or `"inline-end"`.
+ * @prop --calcite-sheet-min-width: Specifies the minimum width of the component When `position` is `"inline-start"` or `"inline-end"`.
+ * @prop --calcite-sheet-height: Specifies the height of the component when `position` is `"block-start"` or `"block-end"`.
+ * @prop --calcite-sheet-max-height: Specifies the maximum height of the component when `position` is `"block-start"` or `"block-end"`.
+ * @prop --calcite-sheet-min-height: Specifies the minimum height of the component when `position` is `"block-start"` or `"block-end"`.
```

Comment:
_Believe_ the original text is in alignment with our doc pattern, where we provide the context of what is needed with "When xyz, then xyz..."